airgun/entities/virtwho_configure.py

```diff
@@ -27,29 +27,24 @@ def _reset_values(self, values):
             vals.update({'hypervisor_type': mapping[hypervisor_value]})
         return vals
 
-    def can_view(self):
-        """Assert if can navigate to virtwhopage"""
+    def permissions(self):
```

Comment:
I think just to avoid confusion can we rename this to check_create_permission and other method to check_update_permission. This may be a little simple to know what method is trying to do from the name.

Reply:
yes, thanks for the suggestion, will modify this with next commit